posthog/api/special_migration.py

```diff
@@ -69,8 +69,23 @@ def force_stop(self, request, **kwargs):
         sm = self.get_object()
         if sm.status != MigrationStatus.Running:
             return response.Response(
-                {"success": False, "error": f"Can't stop a migration that isn't running",}, status=400,
+                {"success": False, "error": f"Can't stop a migration that isn't running.",}, status=400,
             )
         app.control.revoke(sm.celery_task_id, terminate=True)
         process_error(sm, "Force stopped")
         return response.Response({"success": True}, status=200)
+
+    @action(methods=["POST"], detail=True)
+    def force_rollback(self, request, **kwargs):
+        sm = self.get_object()
+        if sm.status != MigrationStatus.CompletedSuccessfully:
+            return response.Response(
+                {
+                    "success": False,
+                    "error": f"Can't force rollback a migration that did not complete successfully. Force stop it instead.",
```

Comment:
Imagine a user wanting to do rollback for whatever state the migration is, e.g. it could be currently running near completion. It would be nice if they could run just use one `rollback --force` like cmd.

Here's how I'd find stop and rollback intuitive:
`stop` just stops doesn't do anything else, with `--rollback` rolls back too (we can default this to true, i.e. one can do `stop --rollback=false` if they don't want the rollback, which we should add as an option for manual rollback actions to be possible)
`rollback` does the roll back if not in completed state & we need `--force` to rollback completed ones too.